cloudbuild-dev.yaml

```diff
@@ -97,7 +97,9 @@ steps:
         "--min-instances",
         "1",
         "--max-instances",
-        "2",
+        "100",
+        "--concurrency",
+        "1",
```

Comment:
Is this needed if concurrency is 1 by default?

Reply:
i guess its there for completeness

Reply:
Technically I can omit it here as it is default by Terraform. But yeah just for completeness purpose